process/plasma_geometry.py

```diff
@@ -249,10 +233,45 @@ def geomty(self):
             thetao,
         )
         physics_variables.sareao = xso
-        physics_variables.sarea = xsi + xso
 
-        #  Cross-sectional area
-        physics_variables.xarea = self.xsecta(xi, thetai, xo, thetao)
+        # icurr = 8 specifies use of the Sauter geometry as well as plasma current.
```

Comment:
Will it ever make sense to use Sauter geometry without using the Sauter plasma current?

Reply:
Ok. If you make this into a new issue we can discuss it there